src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,69 @@
+import classNames from 'classnames';
+import React from 'react';
+import { FilterParam } from '../../types/FilterParam';
+import { Todo } from '../../types/Todo';
+import { checkCompletedTodo, counterOfActiveTodos } from '../../api/todos';
+
+type Props = {
+  todos: Todo[],
+  filterType: FilterParam,
+  setFilterType: (type: FilterParam) => void,
+};
+
+export const Footer: React.FC<Props> = ({
+  todos,
+  filterType,
+  setFilterType,
+}) => {
+  const changedFilter = (event: React.MouseEvent<HTMLAnchorElement>) => {
+    const href = event.currentTarget.getAttribute('href')?.replace('#/', '');
+
+    switch (href) {
+      case 'all':
+        setFilterType(FilterParam.All);
+        break;
+
+      case 'active':
+        setFilterType(FilterParam.Active);
+        break;
+
+      case 'completed':
+        setFilterType(FilterParam.Completed);
+        break;
+
+      default:
```

Comment:
```suggestion
      default:
        break;
```
should be `break` or `return` in any case (including `default`)